src/backend/src/services/projects.services.ts

```diff
@@ -377,6 +379,9 @@ export default class ProjectsService {
           projectNumber: wbsNumber.projectNumber,
```

Comment:
update documentation for this method to specify that it's adding the team to an array of teams, and that it will remove the team if it's already in the array. setProjectTeam is kind of a weird name for this as well, rename if u can think of smthn better

Reply:
I cant think of anything better, maybe addProjectTeam but still its more like a toggle than an add